metadata-ingestion/tests/integration/vertica/test_vertica.py

```diff
@@ -8,26 +8,31 @@
 FROZEN_TIME = "2020-04-14 07:00:00"
 
 
-@freeze_time(FROZEN_TIME)
-@pytest.mark.integration
-@pytest.mark.skip("This does not work yet and needs to be fixed.")
-def test_vertica_ingest(docker_compose_runner, pytestconfig, tmp_path, mock_time):
+def vertica_runner(docker_compose_runner, pytestconfig):
     test_resources_dir = pytestconfig.rootpath / "tests/integration/vertica"
-
     with docker_compose_runner(
         test_resources_dir / "docker-compose.yml", "vertica"
     ) as docker_services:
-        wait_for_port(docker_services, "vertica_ce", 5433, timeout=120)
-
-        # Run the metadata ingestion pipeline.
-        config_file = (test_resources_dir / "vertica_to_file.yml").resolve()
-        run_datahub_cmd(
-            ["ingest", "--strict-warnings", "-c", f"{config_file}"], tmp_path=tmp_path
+        wait_for_port(docker_services, "vertica-ce", 5433)
+        docker_services.wait_until_responsive(
+            timeout=30,
+            pause=1,
         )
+        yield docker_services
 
-        # Verify the output.
-        mce_helpers.check_golden_file(
-            pytestconfig,
-            output_path=tmp_path / "vertica.json",
-            golden_path=test_resources_dir / "vertica_golden.json",
-        )
+
+# Test needs more work to be done , currently it is working fine.
+@freeze_time(FROZEN_TIME)
+@pytest.mark.integration
+def test_vertica_ingest_with_db(pytestconfig, tmp_path):
+    test_resources_dir = pytestconfig.rootpath / "tests/integration/vertica"
+    # Run the metadata ingestion pipeline.
+    config_file = (test_resources_dir / "vertica_to_file.yml").resolve()
```

Comment:
looks like this test doesn't cover models, view lineage, or oauth

Reply:
This test just runs the code and check for errors in vertica.py, we have moved all unit test in dialect . It runs check all features of the vertica plugin

Reply:
https://github.com/vishalkSimplify/sqlalchemy-vertica-dialect/actions/runs/3797696583/jobs/6458860160